### ISSUE TITLE
Animated emote typing preview freezes

### ISSUE THREAD

**Issue by ScrubN** [flex3r/DankChat#471]
The first time an animated emote appears in the typing suggestions list, it seems to always be stuck at frame 0. When typing more to refresh the suggestion list, it will play more of the emote's frames but it will still get stuck before reaching the end and will not loop.

This seems to only affect the typing suggestion list. Animated emotes were playing normally and looping in chat while testing, and the emote menu also works as expected.

**Comment by flex3r** [comment]
Yeah, I've known this issue for a while now. I'm pretty sure it's because of the component I'm using for the suggestions. It's a pretty old one so not sure if this can be fixed with the current one or if a full rewrite might be easier

**Comment by flex3r** [comment]
Fixed with the latest beta
